ddocs/medic-client/views/contacts_by_freetext/map.js

```diff
@@ -1,5 +1,16 @@
 function(doc) {
-  var skip = [ '_id', '_rev', 'type', 'refid' ];
+  var skip = [ '_id', '_rev', 'type', 'refid', 'geolocation' ];
+
+  var usedKeys = [];
+  var emitMaybe = function(key, value) {
+    if (usedKeys.indexOf(key) === -1 && // Not already used
```

Comment:
Yeah, I think it's worthwhile doing that for all views at some point, unless there is a reason why we do that. It seems to just be another thing for CouchDB to parse, AFAICT it doesn't help with anything like making querying easier or anything